ui/app/AppLayouts/Chat/views/ChatMessagesView.qml

```diff
@@ -180,8 +183,13 @@ Item {
         anchors.right: parent.right
         spacing: 0
         verticalLayoutDirection: ListView.BottomToTop
-        cacheBuffer: height * 2 // cache 2 screens worth of items
+        cacheBuffer: height > 0 ? height * 2 : 0 // cache 2 screens worth of items
```

Comment:
wouldn't cacheBuffer: height *2 work as well?

Reply:
Yeah, saw that warning too, good fix!